server/aesdsocket.c

```diff
@@ -212,127 +428,57 @@ int main( int argc, char* argv[]) {
         umask(0);
 
     }
-    
-
-    struct sockaddr_storage client_addr;
-    socklen_t addr_size = sizeof(client_addr);
-    char buffer[BUFFER_SIZE];
-    char sendBuffer[BUFFER_SIZE];
+    pthread_create(&timerStampThread,NULL, timeStamper, NULL);
```

Comment:
@Nicholas-Buckley77 missing a pthread_join for this thread, I think this explains this message:
```
==3446== 272 bytes in 1 blocks are possibly lost in loss record 1 of 1
==3446==    at 0x483DD99: calloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==3446==    by 0x40149CA: allocate_dtv (dl-tls.c:286)
==3446==    by 0x40149CA: _dl_allocate_tls (dl-tls.c:532)
==3446==    by 0x4874322: allocate_stack (allocatestack.c:622)
==3446==    by 0x4874322: pthread_create@@GLIBC_2.2.5 (pthread_create.c:660)
==3446==    by 0x109FC5: main (aesdsocket.c:431)
```